payments/schema/mutations.py

```diff
@@ -333,6 +335,53 @@ def mutate_and_get_payload(cls, root, info, **input):
         return CreateOrderMutation(order=order)
 
 
+class CreateAdditionalProductOrderMutation(graphene.ClientIDMutation):
+    class Input:
+        customer_id = graphene.ID(required=True)
+        lease_id = graphene.ID(required=True)
+        additional_product_id = graphene.ID(required=True)
+
+    order = graphene.Field(OrderNode)
+
+    @classmethod
+    @add_permission_required(Order)
+    @view_permission_required(BerthLease, WinterStorageLease)
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **input):
+        customer_id = input.pop("customer_id")
+        customer = get_node_from_global_id(
+            info, customer_id, ProfileNode, nullable=False
+        )
+
+        additional_product_id = input.pop("additional_product_id", None)
+        additional_product = get_node_from_global_id(
+            info, additional_product_id, AdditionalProductNode, nullable=False
+        )
```

Comment:
No need for `.pop("", None)`. Because it's a required field, it will fail if it doesn't have a value.

```suggestion
        additional_product = get_node_from_global_id(
            info, input.pop("additional_product_id"), AdditionalProductNode, nullable=False
        )
```